src/libmongoc/src/mongoc/mongoc-collection.c

```diff
@@ -1823,6 +1824,11 @@ mongoc_collection_insert_one (
                                        reply,
                                        error,
                                        "insertedCount");
+   
```

Comment:
```suggestion

```

Run clang-format to format code and fix the `clang-format` Evergreen task. To run locally, try:

```bash
./tools/poetry.sh install --with=dev
./tools/poetry.sh run .evergreen/scripts/clang-format-all.sh
```